game.js

```diff
@@ -125,13 +126,13 @@ class Game {
                     && !clock.pause.pause_control[`vacation-${clock.black_player_id}`]
                     && !clock.pause.pause_control[`vacation-${clock.white_player_id}`]) {
 
-                    const forRankedUnranked = getForRankedUnranked(this.state.ranked);
-                    const noPauseMg = `Pausing not allowed ${forRankedUnranked}. Resuming game.`;
-                    this.sendChat(noPauseMg);
-                    if (config.DEBUG) this.log(noPauseMg);
+                    const r_u_strings = get_r_u_strings(this.state.ranked);
```

Comment:
Missing an argument here, but i think the solution would be to remove that argument from that function completely.

Reply:
Scratch that, the solution would be to make a new `get_r_u_string` function with just 1 argument.

Reply:
then i can manually set it to undefined
its really convenient to have one function for all files though, so i'd like to keep it if possible